src/containers/my-cooperations/my-cooperations-details/MyCooperationsDetails.tsx

```diff
@@ -0,0 +1,142 @@
+    defaultResponse: defaultOffer
+  })
+
+  const { offer, price } = response
+
+  const onHandleClick = () => {
+    navigate(
+      createUrlPath(authRoutes.userProfile.path, offer.author._id, {
+        role: UserRoleEnum.Tutor
+      })
+    )
+  }
+
+  const onClickOpenChat = () =>
+    setChatInfo({
+      author: offer.author,
+      authorRole: UserRoleEnum.Tutor,
+      chatId: offer.chatId
+    })
+
+  const languages = offer.languages.map((item: string) => (
+    <Box key={item} sx={style.languageItem}>
+      <DoneIcon color='success' />
+      <Typography>{item}</Typography>
+    </Box>
+  ))
+  return loading ? (
+    <Loader size={50} />
+  ) : (
+    <Box>
+      <Typography sx={style.header}>
+        {t('cooperationDetailsPage.details')}
+      </Typography>
+
+      <Box sx={style.container}>
+        <Typography sx={style.titles}>Cooperations title:</Typography>
+        <Typography sx={style.title}>{offer.title}</Typography>
+        <Typography sx={style.titles}>Tutor:</Typography>
+        <Box>
+          <Box sx={style.profileContainer}>
+            <Avatar
+              src={
+                offer.author.photo &&
+                `${import.meta.env.VITE_APP_IMG_USER_URL}${offer.author.photo}`
+              }
+            />
+            <Typography sx={style.profileName}>
+              {offer.author.firstName} {offer.author.lastName}
+            </Typography>
+            <Typography sx={style.profileDescription}>
+              {offer.author.professionalSummary}
+            </Typography>
+          </Box>
+          <Box>
+            <AppButton
+              onClick={onClickOpenChat}
+              size={SizeEnum.Medium}
+              sx={style.buttons}
+              variant={ButtonVariantEnum.Tonal}
+            >
+              <MessageIcon />
+              {t('common.labels.sendMessage')}
+            </AppButton>
+            <AppButton
+              onClick={onHandleClick}
+              size={SizeEnum.Medium}
+              sx={style.buttons}
+              variant={ButtonVariantEnum.Tonal}
+            >
+              <PersonIcon />
+              {t('cooperationDetailsPage.profile')}
+            </AppButton>
+          </Box>
+        </Box>
+        <Typography sx={style.titles}>Tutoring subject & level:</Typography>
+        <Box sx={style.subjectContainer}>
+          <DesignServicesIcon sx={{ color: offer.category.appearance.color }} />
+          <Typography>{offer.category.name}</Typography>
+          <SubjectLevelChips
+            color={offer.category.appearance.color}
+            proficiencyLevel={offer.proficiencyLevel}
+            subject={offer.subject.name}
+          />
+        </Box>
+        <Typography sx={style.titles}>About cooperation:</Typography>
+        <ShowMoreCollapse
+          collapsedSize={30}
+          collapsedTextLength={100}
+          description={offer.description}
+          sx={style.aboutCooperation}
+          title={''}
+          withoutTitle
```

Comment:
do you need to pass title and withoutTitle props together?

Reply:
no i fixed that